library/x509write_crt.c

```diff
@@ -190,8 +190,8 @@ int mbedtls_x509write_crt_set_subject_alternative_name(mbedtls_x509write_cert *c
                 break;
             }
             default:
-                /* Not supported - skip. */
-                break;
+                /* Not supported - return. */
+                return MBEDTLS_ERR_X509_FEATURE_UNAVAILABLE;
```

Comment:
If we do this for cert we should also adapt csr.
We can do that later, but we need to remember.

I think that in case of generating crt/csr this approach is better as we inform user that operation has failed. Given input is not supported. For parsing I think that skipping is better option.